scripts/e2e/delete_fixtures

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -eu
+
+# Does the inverse of create_fixtures
+echo && echo "[$0] deleting fixtures for api e2e tests..."
+
+delete_doc() {
+	local docUrl="$COUCH_URL/$1"
+	local rev="$(curl $docUrl | jq --raw-output ._rev)"
+	if [[ "$rev" != "null" ]]; then
+		curl -X DELETE "$docUrl?rev=$rev"
+	fi
+}
+
+delete_user() {
+	local username="$1"
+	echo "$API_URL/api/v1/users/$1"
```

Comment:
This looks like debug; should it be removed?  If not, some context would be helpful